integrations/chroma/tests/test_document_store.py

```diff
@@ -128,8 +131,31 @@ def test_from_json(self):
 
     @pytest.mark.integration
     def test_same_collection_name_reinitialization(self):
-        ChromaDocumentStore("test_name")
-        ChromaDocumentStore("test_name")
+        ChromaDocumentStore("test_1")
+        ChromaDocumentStore("test_1")
```

Comment:
why change the collection name for every test?
Is the DB persistent or in-memory?

Reply:
Yes this is interesting, as the DB is in-memory but is retaining the state across different instances. So if I run the tests with same name, they fail because the distance function is set in the first instance.